straxen/analyses/holoviews_waveform_display.py

```diff
@@ -84,15 +84,15 @@ def _records_to_points(*, records, to_pe, t_reference):
         kdims=[hv.Dimension('time', label='Time', unit='sec'),
                hv.Dimension('channel',
                             label='PMT number',
-                            range=(0, straxen.n_tpc_pmts))],
+                            range=(0, config['n_tpc_pmts']))],
         vdims=[hv.Dimension('area', label='Area', unit='pe')])
 
     time_stream = hv.streams.RangeX(source=rec_points)
     return rec_points, time_stream
 
 
 @straxen.mini_analysis(requires=['records'], hv_bokeh=True)
-def hvdisp_plot_records_2d(records, to_pe,
+def hvdisp_plot_records_2d(records, to_pe, config,
```

Comment:
If you are passing on the config, you could be a little lazier and omit the to_pe requirement and simply read it from the config. Might also prevent one from accidently using incompatible options.

Reply:
True, but that would mean adding
```
to_pe = straxen.get_to_pe(run_id, config['gain_model'], config['n_tpc_pmts'])
```
to every mini-analyses that uses `to_pe`. This syntax could also change if we introduce voltage-dependent gain models, since then we'd need an extra input (unless get_to_pe gets special powers, e.g. to fetch from the run doc or slow control info). Currently we have this line in mini_analysis.py, so if you require `to_pe` in a mini-analysis it is just an alias for the line above.